cassandra/deserializers.pyx

```diff
@@ -127,8 +127,8 @@ cdef class DesCounterColumnType(DesLongType):
 
 cdef class DesDateType(Deserializer):
     cdef deserialize(self, Buffer *buf, int protocol_version):
-        cdef double timestamp = unpack_num[int64_t](buf) / 1000.0
-        return datetime_from_timestamp(timestamp)
+        cdef int64_t timestamp_in_us = unpack_num[int64_t](buf) * 1000
```

Comment:
It would work. It also crossed my mind if we shouldn't handle this MS (Cassandra precision) rather than US (Python precision). Since the Python datetime has US support (datetime(year, month, day, hour, minute, second, u)s), I simply decided to go that way.

I really don't have a strong opinion on that. If you prefer I do that change, just confirm here.